.ably/capabilities.yaml

```diff
@@ -0,0 +1,88 @@
+%YAML 1.2
+---
+common-version: 1.2.0-alpha.1
+compliance:
+  Agent Identifier:
+    Agents:
+  Authentication:
+    API Key:
+    Token:
+      Callback:
+      Literal:
+      URL:
+        Query Time:
+  Debugging:
+    Error Information:
+    Logs:
+  Protocol:
+    JSON:
+    MessagePack:
+      .caveats: 'Not supported for realtime'
+  Realtime:
+    Channel:
+      Attach:
+      Subscribe:
+      State Events:
+    Connection:
+      Disconnected Retry Timeout:
+      Lifecycle control:
+      Ping:
+      State Events:
+      Suspended Retry Timeout:
+  REST:
+    Authentication:
+      Authorize:
+      Create Token Request:
+      Get Client Identifier:
+      Request Token:
+    Channel:
+      Encryption:
+      Existence Check:
+      Get:
+      History:
+      Iterate:
+      Name:
+      Presence:
+        History:
+        Member List:
+      Publish:
+        Idempotence:
+      Push Notifications:
+        List Subscriptions:
+        Subscribe:
+      Release:
+      Status:
+        Channel Details: # https://github.com/ably/ably-python/pull/276
```

Comment:
Is there a reason we have the PR link?

Reply:
That's a good question, tbh I just copied the manifest from https://github.com/ably/features/blob/main/sdk-manifests/ably-python.yaml so didn't change any the REST stuff, but I agree we don't need it so i'll remove it now